render/detailed/summary_test.go

```diff
@@ -47,10 +47,10 @@ func TestSummaries(t *testing.T) {
 	// It should summarize nodes' metrics
 	{
 		t1, t2 := mtime.Now().Add(-1*time.Minute), mtime.Now()
-		metric := report.MakeMetric().Add(t1, 1).Add(t2, 2)
+		metric := report.MakeMetric([]report.Sample{{t1, 1}, {t2, 2}})
 		input := fixture.Report.Copy()
 
-		input.Process.Nodes[fixture.ClientProcess1NodeID] = input.Process.Nodes[fixture.ClientProcess1NodeID].WithMetrics(report.Metrics{process.CPUUsage: metric})
+		input.Process.Nodes[fixture.ClientProcess1NodeID].Metrics[process.CPUUsage] = metric
```

Comment:
I don't know how this could work before. `WithMetrics()` causes a merge not a replacement.